docs/operator-manual/application.yaml

```diff
@@ -103,13 +103,32 @@ spec:
         beep: boop-${ARGOCD_APP_REVISION}
       # Toggle which enables/disables env variables substitution in commonAnnotations
       commonAnnotationsEnvsubst: true
+      labelWithoutSelector: false
+      forceCommonLabels: false
+      forceCommonAnnotations: false
       images:
       - gcr.io/heptio-images/ks-guestbook-demo:0.2
       - my-app=gcr.io/my-repo/my-app:0.1
       namespace: custom-namespace
       replicas:
       - name: kustomize-guestbook-ui
         count: 4
+      components:
+        - ../component  # relative to the kustomization.yaml (`source.path`).
+      patches:
+        - target:
+            kind: Deployment
+            name: guestbook-ui
+          patch: |-
+            - op: replace
+              path: /spec/template/spec/containers/0/ports/0/containerPort
+              value: 443
+          patch: |-
+            - op: add
+              path: /spec/template/spec/nodeSelector/
+              value:
+                env: {{.env}}   # patch using attribute from generator
```

Comment:
I could be missing something, but I think go templating is only applicable for ApplicationSets. We should limit this file to Application CR features.

Reply:
Wait, where's the go templating? This is a Kustomize patch.

Reply:
```yaml
                env: {{.env}}   # patch using attribute from generator
```

Looks like they're assuming this is in an AppSet, which is not always the case.

Reply:
Okay, sorry, I have made a mistake, I change it.